server/pulp/server/webservices/views/status.py

```diff
@@ -0,0 +1,43 @@
+"""
+Unauthenticated status API so that other can make sure we're up (to no good).
+"""
+
+from django.views.generic import View
+
+import pulp.server.managers.status as status_manager
+from pulp.server.webservices.views.util import generate_json_response_with_pulp_encoder
+
+
+class StatusView(View):
+    """
+    View for server status
+    """
+
+    def get(self, request):
```

Comment:
Should we require authentication on this view? I don't know for sure one way or another, but if this view causes a lot of queries, it might be a DOS vector.

I assume the old view did not require this, so perhaps we could consider this in a separate effort. What do you think?